packages/docs/components/atoms/Figure/twig-api.md

```diff
@@ -14,18 +14,21 @@ The  [`src`](#src), [`width`](#width) and [`height`](#height) parameters are req
 ### `src`
 
 - Type: `string`
+- Required
 
 Configure the `src` attribute of the image.
 
 ### `width`
 
 - Type: `number`
+- Required
```

Comment:
**detail** width & height have default values `{% set width = width|default(100) %}`, they are not required